client/src/components/table/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import './style.css'
+class Flight extends React.Component{
+  constructor(props){
+    super(props);
+    this.state={  
+      flightNo :'',
+      flightDate:'',
+      flightTime:'',
+      flightDuration:'',
+      flightsArray:[]
+      
+    }
+    this.handleChange=this.handleChange.bind(this);
+    this.flightDate=this.flightDate.bind(this);
+    this.flightTime=this.flightTime.bind(this);
+    this.flightDuration=this.flightDuration.bind(this);
+  } 
+  
+  handleChange=(e)=>{
+    this.setState({flightNo:e.target.value})
```

Comment:
Destructure value from `e.target` then use it as `flightNo` and fix indentation.